### DISCUSSION TITLE
Catupiry

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#309] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/catupiry/

**Comment by maryon-telvanni** [comment]
Detesto catupiry. É... nojento

**Comment by one-eyed-willy** [comment]
> [Maryon Telvanni](</perfil/?user=maryon-telvanni>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/imagens/tirinhas/catupiry/#comment_1113996289>):

> Detesto catupiry. É... nojento

Sou obrigado a concordar

**Comment by brunoaugusto** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/imagens/tirinhas/catupiry/#comment_1114004082>):

> Sou obrigado a concordar

Somos três

**Comment by angellisy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/imagens/tirinhas/catupiry/#comment_1114014786>):

> Somos três

Mas como assim! Catupiry é vida! :sparkling\_heart:
